templates/zerver/help/format-your-message-using-markdown.md

```diff
@@ -72,17 +72,17 @@ A complete list of emojis can be found [here](http://www.webpagefx.com/tools/emo
 
 ## Mentions
 
-To call the attention of another member, you can alert other users by typing `@**username**"`
+To call the attention of another member, you can alert them by typing `@**username**`
 
-If you type `@` and then begin typing their email address or one of their names, and the system will offer you auto-completion suggestions.
+If you type `@` and then begin typing their email address or one of their names, the system will offer you auto-completion suggestions.
```

Comment:
I could change this to make it a bit clearer, even though I think "their" is still correct.